### PR TITLE
JENKINS-33805 - rename install/upgrade files to be more clear & consistent

### PR DESCRIPTION
**Pull request by kzantow**
Renamed the install and upgrade files to be somewhat more clear & consistent.

Addresses: https://issues.jenkins-ci.org/browse/JENKINS-33805

@reviewbybees esp. @daniel-beck

### CODE REVIEW

File: core/src/main/java/jenkins/install/InstallUtil.java [hunk 1]
```diff
@@ -183,15 +183,15 @@ static void saveLastExecVersion(@Nonnull String version) {
     }
 
     static File getConfigFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), "config.xml");
+        return new File(Jenkins.getInstance().getRootDir(), "config.xml");
     }
 
     static File getLastExecVersionFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), ".last_exec_version");
+        return new File(Jenkins.getInstance().getRootDir(), "last_run_version");
```
Comment (daniel-beck) [inline core/src/main/java/jenkins/install/InstallUtil.java]: By convention, files in Jenkins home indicate what they were based on class names. Maybe `jenkins.install.InstallUtil.lastExecVersion`?

File: core/src/main/java/jenkins/install/UpgradeWizard.java [hunk 2]
```diff
@@ -47,7 +47,7 @@
      * This file records the vesrion number that the installation has upgraded to.
      */
     /*package*/ File getStateFile() {
-        return new File(Jenkins.getInstance().getRootDir(),"upgraded");
+        return new File(Jenkins.getInstance().getRootDir(),"last_upgrade_version");
```
Comment (daniel-beck) [inline core/src/main/java/jenkins/install/UpgradeWizard.java]: `jenkins.install.UpgradeWizard.state`?

File: core/src/main/java/jenkins/install/InstallUtil.java [hunk 3]
```diff
@@ -183,15 +183,15 @@ static void saveLastExecVersion(@Nonnull String version) {
     }
 
     static File getConfigFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), "config.xml");
+        return new File(Jenkins.getInstance().getRootDir(), "config.xml");
     }
 
     static File getLastExecVersionFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), ".last_exec_version");
+        return new File(Jenkins.getInstance().getRootDir(), "jenkins.install.InstallUtil.lastExecVersion");
```
Comment (batmat) [inline core/src/main/java/jenkins/install/InstallUtil.java]: To be a wee bit more consistent with the (seemingly?) common pattern in the rest of the codebase, use `InstallUtil.class.getName()+"."+lastExecVersion` instead?

Comment (kzantow) [inline core/src/main/java/jenkins/install/InstallUtil.java]: @batmat intentionally didn't do that in the case of future refactoring (which is likely to happen based on open JIRA issues)

Comment (batmat) [inline core/src/main/java/jenkins/install/InstallUtil.java]: ack :ok_hand:

**Comment by daniel-beck** [comment]
:bee: Looks good to me.

**Comment by recena** [comment]
@kzantow The idea is, one file per variable or param to check?

File: core/src/main/java/jenkins/install/InstallUtil.java (hunk 3 above)
Comment (recena) [inline core/src/main/java/jenkins/install/InstallUtil.java]: I agree with @kzantow though I'd like to use some convention clear and documented for this kind of marker-files.

**Comment by recena** [comment]
Me too, :bee: LGTM

**Comment by kzantow** [comment]
@recena no changes here except for renaming the files, they were just named somewhat randomly

File: core/src/main/java/jenkins/install/UpgradeWizard.java [hunk 4]
```diff
@@ -47,7 +47,7 @@
      * This file records the vesrion number that the installation has upgraded to.
      */
     /*package*/ File getStateFile() {
-        return new File(Jenkins.getInstance().getRootDir(),"upgraded");
+        return new File(Jenkins.getInstance().getRootDir(),"jenkins.install.UpgradeWizard.state");
```
Comment (recena) [inline core/src/main/java/jenkins/install/UpgradeWizard.java]: You know, a space after `,` :smile:

**Comment by recena** [comment]
@kzantow Did you consider to store all of these files in a specific folder?

**Comment by kzantow** [comment]
@recena sorta, I think it would be better for this to be a single file containing an upgrade history, not important enough for 2.0: https://issues.jenkins-ci.org/browse/JENKINS-34006

**Pull request merged**